egs/wsj/s5/steps/nnet3/make_cnn_tdnn_configs.py

```diff
@@ -0,0 +1,397 @@
+#!/usr/bin/env python
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import re, os, argparse, sys, math, warnings
+import numpy as np
```

Comment:
BTW I don't think numpy is critical here. We usually avoid importing numpy unless it is absolutely necessary. Could you replace your 2D-numpy arrays with list of lists ?